packages/@webex/plugin-meetings/src/webinar/collection.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) 2015-2023 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import AmpCollection from 'ampersand-collection';
```

Comment:
why use ampersand collection here and not just a object map ?

Reply:
its not being maintained

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
@arun3528  Please help review it again, thanks